### PR TITLE
Select all check box correction

### PR DESCRIPTION
**Pull request by geetanshu-m**
Pull Request for Issue #452 .

### Summary of Changes

Select all checkbox is now checked when all the items are selected.

### CODE REVIEW

**Comment by kasvith** [comment]
Seems we've get rid of confusion :)

File: administrator/components/com_media/resources/scripts/components/toolbar/toolbar.vue [hunk 1]
```diff
@@ -51,7 +51,7 @@
                 return (this.isGridView) ? 'fa fa-list' : 'fa fa-th';
             },
             toggleSelectAllBtnIcon() {
-                return (this.allItemsSelected) ? 'fa fa fa-square-o' : 'fa fa-check-square-o'
+                return (this.allItemsSelected) ? 'fa fa-check-square-o' : 'fa fa fa-square-o'
```
Comment (kasvith) [inline administrator/components/com_media/resources/scripts/components/toolbar/toolbar.vue]: why two `fa`

Comment (geetanshu-m) [reply administrator/components/com_media/resources/scripts/components/toolbar/toolbar.vue]: I just exchanged ```fa fa fa-square-o``` and ```fa fa-check-square-o``` 
do one ```fa``` need to be removed?

Comment (kasvith) [reply administrator/components/com_media/resources/scripts/components/toolbar/toolbar.vue]: @ciar4n

**Comment by laoneo** [comment]
If I'm not wrong we were discussing that at the last code sprint and decided to go the way how it now.

**Comment by ciar4n** [comment]
I think it has changed a couple of times. Personally I agree with this PR. Makes sense that the icon represents the current state.

**Comment by kasvith** [comment]
Makes sense for me too

**Comment by dneukirchen** [comment]
For me too. It was confusing before. Thx @proman24

**Comment by kasvith** [comment]
thanks @proman24

**Pull request merged**